docs/02-app/01-building-your-application/11-upgrading/05-from-create-react-app.mdx

```diff
@@ -64,55 +64,13 @@ Create a `next.config.mjs` at the root of your project. This file will hold your
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   output: 'export', // Outputs a Single-Page Application (SPA).
-  distDir: './dist', // Changes the build output directory to `./dist/`.
+  distDir: './build', // Changes the build output directory to `./dist`.
```

Comment:
./build/` is the default build output directory on CRA. We don't want to change it in case some of the user's tools and processes (e.g., deployment pipelines) depend on that directory.